src/landing-page/landing-page.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { useState, createInterpolateElement } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import {
+	Button,
+	ExternalLink,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { fetchThemeStyleData, downloadExportedTheme } from '../resolvers';
+import { downloadFile } from '../utils';
+import { CreateThemeModal } from './create-modal';
+
+export default function LandingPage() {
+	const [ themeStyleData, setThemeStyleData ] = useState( '' );
+	const [ createModalType, setCreateModalType ] = useState( false );
+
+	useSelect( async () => {
+		setThemeStyleData( await fetchThemeStyleData() );
```

Comment:
I don't think we need to add a custom endpoint since all we need is the theme name. How about just the code below?

```javascript
import { store as coreStore } from '@wordpress/core-data';

export default function LandingPage() {
	// ...

	const themeName = useSelect( ( select ) =>
		select( coreStore ).getCurrentTheme()
	)?.name?.raw;

	// ...
}

Reply:
I don't believe that we have access to the coreStore outside of the Editor... I might be wrong, but I think that this was giving some issues last time I tried to build an interface in this way.